niet/actions.py

```diff
@@ -0,0 +1,36 @@
+def delete(input_dict, search):
+
+    if type(search) == list:
+        for path in search:
+            input_dict = delete(input_dict, path)
+    else:
+
+        # Remove whole dict
+        if search == ".":
+            input_dict = {}
```

Comment:
Are you sure we want to continue the execution after this?

I mean maybe just return the dict is enough there `return {}`